app/controllers/user/shares_controller.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+class User::SharesController < User::BaseController
+  before_action :set_share, only: %i[show edit update destroy]
+
+  # GET /shares
+  def index
+    @shares = current_user.shares
+  end
+
+  # GET /shares/1
+  def show; end
+
+  # GET /shares/new
+  def new
+    # pre-fill any supplied params (e.g. birth_record if creating from the birth
+    # record page)
+    @share = if params.keys.include? 'share'
+               User::Share.new(share_params)
+             else
+               User::Share.new
+             end
+    @organisations = OrganisationUser.all
```

Comment:
have you considered making this a list of organisations, rather than users.. with OrganisationUser belonging to an Organisation.

Reply:
My working assumption is that ECEs have an organisational account and individual staff won't have their own logins. That's not necessarily true but it's what I understand happens in the MoE authentication server the SMSs use so I think it's interesting to simulate.